Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -36,6 +34,7 @@ class XcodeSpec: QuickSpec {
 		}
 
 		describe("determingSwiftInformation") {
+			let currentSwiftVersion = swiftVersion.single()?.value
```

Comment:
`currentSwiftVersion` should probably be non-optional. Otherwise if the command failed for some reason, it could mask a failure when we compare below.